toolchain/sem_ir/stringify_type.cpp

```diff
@@ -372,6 +373,13 @@ auto StringifyTypeExpr(const SemIR::File& sem_ir, InstId outer_inst_id)
       case CARBON_KIND(ImplWitnessAccess inst): {
         auto witness_inst_id =
             sem_ir.constant_values().GetConstantInstId(inst.witness_id);
+        if (auto symbolic_witness =
+                sem_ir.insts().TryGetAs<SemIR::ImplSymbolicWitness>(
```

Comment:
I think you can avoid using the std::optional conversion to query the kind. `TryGetAs` is primarily intended for when you're making use of the result.

```suggestion
        if (sem_ir.insts().Is<SemIR::ImplSymbolicWitness>(
```

Reply:
Yeah, I think we may need it to address the TODO, but good point as I didn't use anything yet.